src/vs/workbench/contrib/webview/browser/webviewElement.ts

```diff
@@ -506,6 +505,10 @@ export class WebviewElement extends Disposable implements IWebview, WebviewFindD
 
 				this._messagePort = e.ports[0];
 				this._messagePort.onmessage = (e) => {
+					if (e.data.channel === 'did-context-menu') {
```

Comment:
You can use the existing `did-context-menu` listener on line 246 for this

Reply:
Sure will check and update

Reply:
Done @mjbvz 🙂